python/kserve/kserve/grpc/grpc_predict_v2.proto

```diff
@@ -0,0 +1,362 @@
+// Copyright 2022 The KServe Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
```

Comment:
just curious, why is this different from https://github.com/kserve/kserve/blob/master/docs/predict-api/v2/grpc_predict_v2.proto ?

Reply:
The one in docs directory does not include the model repository extension for load/unload apis